tensorflow/python/compiler/tensorrt/__init__.py

```diff
@@ -15,5 +15,69 @@
 """Exposes the python wrapper for TensorRT graph transforms."""
 
 # pylint: disable=unused-import,line-too-long
+import os
+import sys
+from tensorflow.compiler.tf2tensorrt import _pywrap_py_utils
 from tensorflow.python.compiler.tensorrt import trt_convert as trt
+from tensorflow.python.compiler.tensorrt import utils as trt_utils
+from tensorflow.python.platform import tf_logging as logging
 # pylint: enable=unused-import,line-too-long
+
+
+def _raise_trt_version_not_supported(version_type, trt_version):
+  assert version_type in ["linked", "loaded"], (
+    f"Incorrect value received for version_type: {version_type}. " \
+    f"Accepted: ['linked', 'loaded']"
+  )
+
+  logging.error(
+      f"The support for {version_type} version of TensorRT: "\
+      f"`{trt_utils.version_tuple_to_string(trt_version)}` has now been " \
+      f"removed. Please upgrade to TensorRT 7 or more recent.")
+
+  raise RuntimeError(f"Incompatible {version_type} TensorRT versions")
+
+
+if not _pywrap_py_utils.is_tensorrt_enabled():
+  logging.error(
+      "Tensorflow needs to be built with TensorRT support enabled to allow "
+      "TF-TRT to operate.")
+
+  raise RuntimeError("Tensorflow has not been built with TensorRT support.")
+
+
+# Do not execute the following on a machine that is not linux-based and during
+# bazel build execution.
+if "bazel" not in os.environ["_"] and sys.platform == "linux":
```

Comment:
You can do "if not build: realy return" to reduce the needed indentation for the code.